script/create_scan_project_in_ui.rb

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../config/environment'
+
+class CreateScanProjectInUi
+  include JWTHelper
+
+  def initialize
+    @filename = 'vendor/active_projects.csv'
+    @jwt = build_jwt(Account.hamster.login)
+    @headers = %w[name repo_url]
+  end
+
+  def execute
+    CSV.foreach(@filename, headers: true) do |row|
+      params = { JWT: @jwt, name: row['name'], repo_url: fix_url(row['repo_url']), license_name: row['license_name'] }
+      project = create_scan_project(params)
+
+      next if project.nil?
+
+      if project && (project['name'].to_s.include?("can't be blank") || project['enlistments.base'])
+        insert_failed_url_data(row)
+      end
+      create_code_location_scan(project, row) if project && project['id']
+    end
+  end
+
+  private
+
+  def create_scan_project(params)
+    uri = URI("#{ENV['URL_HOST']}/api/v1/projects.json")
+    response = Net::HTTP.post_form(uri, params)
+    JSON.parse(response.body)
+  rescue StandardError => e
+    puts e.message
+  end
+
+  def fix_url(url)
+    new_url = url.strip.chomp('/')
+    return new_url unless url.match('github.com')
+
+    new_url.sub(/\.git$/, '')
+           .sub(%r{^git://}, 'https://')
+           .sub(%r{^git@github.com:}, 'https://github.com/')
```

Comment:
This operation is already done in Fisbot's **code_locations_controller**. Any specific reason to cleanup url here?

Reply:
CSV file is having URL which needs to be fixed before making calls. So I am using the same functionality here also.

Reply:
What is the reason for CSV file urls to be cleaned before making the api call? Is it a requirement we specified somewhere?

Reply:
If we are not cleaning the URL before making calls, it is not fetching the value from the repo and hence in the forge method, we are getting an empty result. We are not able to create a project.